packages/@ionic/cli/src/commands/deploy/manifest.ts

```diff
@@ -83,4 +100,53 @@ export class DeployManifestCommand extends DeployCoreCommand {
       })
       .join(' ');
   }
+
+  private getCapacitorConfigJsonPath(): string {
+    return path.resolve(this.env.ctx.execPath, CAPACITOR_CONFIG_JSON_FILE);
+  }
+
+  private getCapacitorCLIConfig = lodash.memoize(async (): Promise<CapacitorCLIConfig | undefined> => {
+    const args = ['config', '--json'];
+    const log = new Logger({
+      level: LOGGER_LEVELS.INFO,
+      handlers: createDefaultLoggerHandlers(),
+    });
+    const shell = new Shell({ log }, { alterPath: p => { return prependNodeModulesBinToPath(this.env.ctx.execPath, p)} });
```

Comment:
I had to create a new shell to force `prependNodeModulesBinToPath`. If `ionic.config.json` is not present, then `this.env.shell` will not implement this, and the Capacitor command will fail.

Reply:
still, put this as comment in the code

Reply:
Good idea, done.